WolvenKit.App/Models/MaterialXmlModel.cs

```diff
@@ -0,0 +1,243 @@
+﻿using System.Xml.Serialization;
+
+namespace WolvenKit.App.Models;
+
+// NOTE: Generated code may require at least .NET Framework 4.5 or .NET Core/Standard 2.0.
```

Comment:
Not sure about this, but since it probably doesn't change that often (won't be regenerated) maybe we could/should apply naming conventions?